eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -40,6 +40,7 @@ jobs:
       - template: ../../../common/pipelines/templates/steps/verify-links.yml
         parameters:
           Directory: ''
+          CheckLinkGuidance: $true
```

Comment:
Can we also add this to the aggregate reports pipeline and queue that build to verify all the links in the repo?

Reply:
Nevermind this is the android repo which doesn't have that check.